### DISCUSSION TITLE
Is there a way to get more detailed information about what happened in file-enumerator?

### DISCUSSION THREAD

**Discussion by Gundam98** [eslint/eslint#17185] [Q&A]
According to the debug output, the file-enumerator module runs pretty slow while linting.
Is there any way to get more logs about the file-enumerator module?

**Comment by sam3k** [comment]
Hi @Gundam98, for those of us not familiar with all the ESlint repos; could you point us to the right repo and if possible provide a reproducible demo with installation steps?

**Reply by Gundam98** [reply]
Hi @sam3k, after some more investigating, I found that it was caused by @eslint-typescript/parser, which slowed down the eslint execution. Thank you for your prompt reply.
